src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -750,6 +766,45 @@ public function links()
     }
 
     /**
+     * Returns an Image object for the first node in the list.
+     *
+     * @param string $method The method for the image (get by default)
+     *
+     * @return Image An Image instance
+     *
+     * @throws \InvalidArgumentException If the current node list is empty
+     *
+     * @api
+     */
+    public function image($method = 'get')
```

Comment:
Yes, remove ``$method`` entirely.

Well, I would overwrite the constructor for the Image class so that it does not have the method argument at all (and btw, it is an optional argument in AbstractUriElement).

I don't even understand why we have a ``$method`` argument in ``->link()`` btw. A link is always GET in HTML